function-session/src/main/java/com/future/function/session/bean/AuditorAwareWrapper.java

```diff
@@ -10,9 +10,13 @@
   @Override
   public String getCurrentAuditor() {
 
+  try {
```

Comment:
Emm why does this now have default to 'system'?

Reply:
because if not, the mongock can't create the first data of user -> admin, and will fail the migration. any ideas?

Reply:
Not at the moment, but I think this could cause problems for other unidentified user's operations on the database, though it should not happen actually.

cc @priagungs @RickyKennedy25 for any ideas